content/network/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Network of Research Software Quality
+description: Network Information
+identifier: Network
+weight: 10
+showDate: false
+groupByYear: false
+---
+
+## Network of Research Software Quality ##
+
+The EVERSE project is committed to improving the quality of software in European research in the international context. EVERSE will launch a Network of Research Software Quality to allow us to work with individuals and partner organisations to achieve this common goal.
+
+### Vision ###
+Our vision of the EVERSE Network of Research Software Quality is to establish a Community of Practice around improving the quality of research software in Europe and beyond. EVERSE and partners work to provide standard and documented practices around tools and training for software developers, researchers who code and research software engineers and service providers, to help them write better code. We help define the meaning and principles of software quality for the research domain. We work for enhanced recognition for software developers contributing to research activities.
+
+[//]: <> (add picture/banner?)
```

Comment:
Yes! The page needs it to break it up a bit. You could try using the flex-content or balanced-content codes to get the image and text nicely arranged next to each other.